### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by jakejarrett**
* Added flex to mainview & made The max-width 100% (Allows more content to fit on wider screens)
* Moved Search to the top panel/bar
* Changed color for search box to increase contrast
* added ```-webkit-user-drag: none;``` to img

*atom for some reason wanted to redo every end line on save*

# Preview
soundnode

EDIT: for some reason it wouldn't let me target soundnode:develop

### CODE REVIEW

**Comment by weblancaster** [comment]
Thanks @jakejarrett 

I'm updating to the latest version of nw.js and as soon I finish that I will review this one.

**Comment by MichielDeMey** [comment]
> Added flex to mainview

I would add it to the `ul.songlist` element, that way you can utilise `space-around` which will look really nice. (notice the even spacing)



```
.songList {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-around;
}
```

> atom for some reason wanted to redo every end line on save

Don't worry, it's only removing trailing spaces and adding appropriate newlines which is a good thing.

**Comment by weblancaster** [comment]
I agree with @MichielDeMey 

Let us know when you do that review @jakejarrett 

Thanks guys

**Comment by jakejarrett** [comment]
Cheers haha, That allows the items to fill up the bottom row if only 3 are loaded, Looks a lot nicer. :+1:

File: app/public/stylesheets/sass/_components/_view-container.scss [hunk 1]
```diff
@@ -34,4 +34,9 @@
...
       margin-right: 0;
     }
-}
\ No newline at end of file
+}
+
+.streamView_inner {
+  margin-top: 50px;
+  margin-bottom: -50px;
+}
```
Comment (MichielDeMey) [inline app/public/stylesheets/sass/_components/_view-container.scss]: I'm generally not a big fan of negative margin/padding hacks.
Any particular reason we need this?

Comment (jakejarrett) [inline app/public/stylesheets/sass/_components/_view-container.scss]: oops. i must have accidentally left that in. I was testing to see if that did anything to the second row.

Comment (weblancaster) [inline app/public/stylesheets/sass/_components/_view-container.scss]: Haha thanks for the reviewing.. yeah seemed weird `margin-top: 50` and `margin-bottom: -50`

**Comment by weblancaster** [comment]
Hi @jakejarrett I was reviewing your improvements and I found some things that need to be fixed before I merge into master.

1. the search box is overlapping the logo in the header when the window isn't wide enough
2. theres a js error on the console.

screen shot 2015-04-14 at 11 51 03 am

**Comment by jakejarrett** [comment]
I was unable to see any js errors from my end. When i tried liking songs, it would 404 with the soundcloud api though.

**Comment by MichielDeMey** [comment]
I doubt the JS error was introduced in this PR since no JS files were touched.
The first issue still stands though, the logo will overlap the searchable when resized.

**Comment by weblancaster** [comment]
@jakejarrett @MichielDeMey I tried again and I did not see any js error.. my bad.

**Comment by jakejarrett** [comment]
I found out that my package.json had a min-width of 1152px for the app, which looked fine for me. But This *should* fix it. :+1:

**Comment by weblancaster** [comment]
Thanks @jakejarrett I'm going to update my local and test again just to make sure, but seems like things are good.

**Comment by weblancaster** [comment]
@jakejarrett can you remove the header logo.. let's just get rid of that.

then can you rebase and push the commit please.

Thanks

**Comment by MichielDeMey** [comment]
Simply sqaush all the commits to a single commit.
I use [SourceTree](http://www.sourcetreeapp.com/) to do this, or check out http://davidwalsh.name/squash-commits-git

**Comment by jakejarrett** [comment]
Haha thanks @MichielDeMey I will check sourcetree out, the github client doesn't really help with squashing.

**Comment by weblancaster** [comment]
@jakejarrett you can do `git reset --soft HEAD~1`

Change the number 1 to the number of your commits then you just need to commit again and everything will be in one commit.

**Comment by jakejarrett** [comment]
Cheers, Cleaned my mess up haha. :+1: I must have messed the git file up by accident.

**Comment by weblancaster** [comment]
I'm going to review it locally tonight. Thanks for the great work @jakejarrett

**Comment by weblancaster** [comment]
Hey @jakejarrett sorry I did not have the time to stop and review this on my machine.. Hopefully this weekend I will make the time to do it.

Thanks.

**Comment by weblancaster** [comment]
Thanks for working on this @jakejarrett

**Pull request merged**